demo/billing/src/mz_client.rs

```diff
@@ -126,7 +126,7 @@ impl MzClient {
 
         let query = format!(
             "CREATE SINK {sink} FROM billing_monthly_statement INTO KAFKA BROKER '{kafka_url}' TOPIC '{topic}' \
-             FORMAT AVRO USING CONFLUENT SCHEMA REGISTRY '{schema_registry}'",
+             WITH (consistency = true) FORMAT AVRO USING CONFLUENT SCHEMA REGISTRY '{schema_registry}'",
```

Comment:
We intentionally don't want the user to choose their own consistency topic name?

Reply:
hmmm the user already doesn't have a lot of leeway in the produced topic name because if you pass in `foo` as the sink topic prefix, the generated topic will have a name like `foo-{global_id}-{startup-time}-{nonce}`

i wanted to control consistency topic name creation to a. make reasoning about this easier and b. help the user not fall into the debezium mindset where the same consistency topic is used across multiple tables. The way that its set up it should be more obvious that the topic `foo-{global_id}-{startup-time}-{nonce}-consistency` only has consistency info for `foo`. wdyt?